flags/custom-ops.json

```diff
@@ -38,7 +38,14 @@
               {
                 "ends_with": [{"var": "id"}, "xyz"]
               },
-              "postfix", "none"
+              "postfix", {
+                "if": [
+                  {
+                    "ends_with": [{"var": "id"}, 3]
+                  },
+                  "fail", "none"
+                ]
+              }
```

Comment:
Currently this works `3 ends_with 3` is falsy when they are both ints. We can decide if that's actually what we want.

Reply:
I could add that to my pending changes [here](https://github.com/open-feature/flagd/pull/1041).